components/OwlCarousel.jsx

```diff
@@ -141,7 +140,7 @@ class OwlCarousel extends React.Component {
 	}
 
 	componentDidMount() {
-		this.$node = $(findDOMNode(this));
+		require('../src/owl.carousel');
```

Comment:
Must be here to allow server-side rendering. Otherwise, you get the `window is not defined` error we all hold most dear.